js/main.js

```diff
@@ -18,12 +21,98 @@ function resetForm(event) {
+
+  document.querySelector('#form-image').setAttribute('src', 'images/placeholder-image-square.jpg');
 
 }
 
 $submit.addEventListener('submit', resetForm);
+// New entry *END*
+
+// For Dom Creation *START*
+function newEntryCreation(id) {
+  var listItem = document.createElement('li');
+  listItem.className = 'entry-bottom-margin';
+  var listFlex = document.createElement('div');
+  listFlex.className = 'justify-between row';
+  var entryImage = document.createElement('img');
+  entryImage.className = 'column-half image-padding square-image image-bottom-margin';
+  var columnHalfDiv = document.createElement('div');
+  columnHalfDiv.className = 'column-half';
+  var entryTitle = document.createElement('h3');
+  entryTitle.className = 'entry-title sans-serif input-title-block-margin-reset title-margin';
+  var entryNotes = document.createElement('p');
+  entryNotes.className = 'input-font-size sans-serif';
+
+  entryImage.setAttribute('src', data.entries[id].photoURL);
+  entryTitle.textContent = data.entries[id].title;
+  entryNotes.textContent = data.entries[id].notes;
+
+  listItem.appendChild(listFlex);
+  listFlex.appendChild(entryImage);
+  listFlex.appendChild(columnHalfDiv);
+  columnHalfDiv.appendChild(entryTitle);
+  columnHalfDiv.appendChild(entryNotes);
+
+  return listItem;
+}
+
+function loadPage(event) {
+  var entryList = document.querySelector('ul');
+  for (var i = data.entries.length - 1; i >= 0; i--) {
+    entryList.appendChild(newEntryCreation(i));
+  }
+  var noEntries = document.querySelector('[name="no-entry"');
+  if (data.entries.length !== 0) {
+    noEntries.className = 'column-full sans-serif font-size row justify-center align-center hidden';
+  }
+
+}
+window.addEventListener('DOMContentLoaded', loadPage);
+
+// DOM Creation *END*
+
+// Switch Tabs *START*
+function tabView(event) {
+  event.preventDefault();
+  var dataView = event.target.getAttribute('data-view');
+  var switching = document.querySelectorAll('.tab-view');
+  for (var i = 0; i < switching.length; i++) {
+    if (switching[i].getAttribute('data-view') === dataView) {
+      switching[i].className = 'tab-view';
+    } else {
+      switching[i].className = 'tab-view hidden';
+    }
+  }
+}
+var entriesLink = document.querySelector('.tab');
+var newEntry = document.querySelector('.new');
+entriesLink.addEventListener('click', tabView);
+newEntry.addEventListener('click', tabView);
+// Switching Tabs *END*
+
+// New entry appearance and switching tabs after submitting
+//* START*
+function newAddition(event) {
+  var entryList = document.querySelector('ul');
+  entryList.prepend(newEntryCreation(data.entries.length - 1));
+  var switching = document.querySelectorAll('.tab-view');
+  for (var i = 0; i < switching.length; i++) {
+    if (switching[i].className === 'tab-view') {
+      switching[i].className = 'tab-view hidden';
+    } else {
+      switching[i].className = 'tab-view';
+    }
+  }
```

Comment:
You repeat this sort of block of code in a couple of places, it would be better to make this its own function. named something like `switchView` that takes an argument of the name of the view to switch to.